project/RootBuild.scala

```diff
@@ -191,6 +191,7 @@ object RootBuild extends Build {
       "org.slf4j" % "slf4j-api" % slf4jVersion,
       "org.slf4j" % "slf4j-log4j12" % slf4jVersion,
       "commons-configuration" % "commons-configuration" % "1.6",
+      "com.google.api-client" % "api-client" % "1.18.0-rc",
```

Comment:
@ctn Yup. thank you! I actually had the wrong artifact name, should be google-api-client